installers/MacInstaller.hs

```diff
@@ -46,7 +46,7 @@ main = do
   writeFile (dir <> "/Daedalus") $ unlines
     [ "#!/usr/bin/env bash"
     , "cd \"$(dirname $0)\""
-    , "mkdir -p \"$HOME/Library/Application Support/Daedalus/\"{Wallet-0.2,DB-0.2,Logs,Secrets}"
+    , "mkdir -p \"$HOME/Library/Application Support/Daedalus/\"{Secrets-0.5}"
```

Comment:
Lesson for future. A bash "gotcha" apparently is that a singleton list of {}-expansions does not get expanded, it turns into literal {}s.

    $ echo foo{bar}
    foo{bar}
    $ echo foo{bar,baz}
    foobar foobaz